.github/workflows/rust.yml

```diff
@@ -1,13 +1,20 @@
-name: Rust CI
+name: Rust CI/CD
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
 
 env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  build:
-    name: Build and Test
+  # Continuous Integration Job
+  ci:
+    name: Continuous Integration
```

Comment:
Why do you change the name here? Keep the previous name. CI means nothing, it is a process not a step